src/appengine/handlers/cron/bandit_probabilities.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2019 Google LLC
```

Comment:
bandit_probabilities.py should be renamed. Maybe something like fuzzing_strategy_experiments.py. This should be much clearer for someone without context who is looking over the files in the repo, and would differentiate this file from any others where we decide to run multi-armed bandit experiments in the future.